schema-source/ASHRAE229_extra.schema.yaml

```diff
@@ -4884,15 +4884,15 @@ Chiller:
       AppG Used By TCDs: True
       AppG P_RMD Equals U_RMD: True
       AppG B_RMD Equals P_RMD: True
-      AppG B_RMDs Same: "unknown"
+      AppG B_RMDs Same: True
```

Comment:
The number varies based on capacity which could change across rotations. I think this should be false then.